app-backend/backsplash-core/src/main/scala/com/rasterfoundry/backsplash/MosaicImplicits.scala

```diff
@@ -386,6 +386,7 @@ class MosaicImplicits[HistStore: HistogramStore](histStore: HistStore)
 
   implicit val mosaicHasRasterExtents: HasRasterExtents[BacksplashMosaic] =
     new HasRasterExtents[BacksplashMosaic] {
+      @SuppressWarnings(Array("TraversableHead"))
```

Comment:
I think instead of suppressing the warning, we should attempt to convert to a NEL at the end, and throw a more helpful error-message there -- as it stands, if we happen to get an empty list of extents (no idea how that would happen, but anyway), we'll get a "head of empty list", which will get transformed into an `UnknownException` and won't help us much to figure out what went wrong.